content/docs/render-props.md

```diff
@@ -50,11 +50,11 @@ class MouseTracker extends React.Component {
 }
 ```
 
-As the cursor moves around the screen, the component displays its (x, y) coordinates in a `<p>`.
+Fare ekranda hareket ettikçe, bileşen (x,y) koordinatlarını `<p>` içerisinde gösterir.
 
-Now the question is: How can we reuse this behavior in another component? In other words, if another component needs to know about the cursor position, can we encapsulate that behavior so that we can easily share it with that component?
+Şimdi soru şu: Bu davranışı başka bir bileşen içerisinde nasıl tekrar kullanabiliriz? Diğer bir deyişle, Başka bir bileşen farenin pozisyonunu bilmek isterse bu davranışı rahatça bu bileşenle paylaşmak için encapsulate edebilir miyiz?
 
-Since components are the basic unit of code reuse in React, let's try refactoring the code a bit to use a `<Mouse>` component that encapsulates the behavior we need to reuse elsewhere.
+Bileşenler React’ın ana kod yeniden kullanım ünitesi olduğundan, istediğimiz davranışı başka bir yerde kullanabilmemiz için `<Mouse>` bileşeninin kodunu biraz değiştirmeyi deneyelim.
```

Comment:
What about `Bileşenler React'te yeniden kod kullanımının temel birimi olduğundan,...` ?

Reply:
Yeah this seems a lot better, I will change it.